jira-slack-server-integration/jira-slack-server-integration-plugin/src/main/java/com/atlassian/jira/plugins/slack/web/actions/MigrateToSlackAction.java

```diff
@@ -9,6 +11,7 @@
 /**
  * Provides context for migrate to Slack channel selection dialog.
  */
+@SupportedMethods(RequestMethod.GET)
```

Comment:
Migration features I think could be removed since HipChat isn't a thing anymore. For context:
- There was a script that would inject a migration button in HipChat configuration page
- And the endpoints that would receive the data for the migration
- I think there's also a dialog.

No need to tackle it in this PR, I'm just mentioning it because it would be one less thing to maintain.